dev/merge_spark_pr.py

```diff
@@ -538,6 +538,9 @@ def initialize_jira():
             else:
                 raise e
     elif JIRA_USERNAME and JIRA_PASSWORD:
+        print("You can use JIRA_ACCESS_TOKEN instead of JIRA_USERNAME/JIRA_PASSWORD.")
+        print("Visit https://issues.apache.org/jira/secure/ViewProfile.jspa ")
+        print("and click 'Personal Access Tokens' menu to manage your own tokens.")
```

Comment:
We can add a message to notify the maintainer to update Jira to the newest version.  If their Jira library is outdated, the token_auth may not function properly.

Reply:
I'll set it to the latest version because we didn't had the minimum before.
- https://github.com/pycontribs/jira/releases/tag/3.5.2